client.go

```diff
@@ -117,8 +118,9 @@ func NewClient(addrs []string, conf *Config) (Client, error) {
 		cachedPartitionsResults: make(map[string][maxPartitionIndex][]int32),
 		coordinators:            make(map[string]int32),
 	}
-	for _, addr := range addrs {
-		client.seedBrokers = append(client.seedBrokers, NewBroker(addr))
+
+	for _, index := range rand.Perm(len(addrs)) {
```

Comment:
we need to seed the random number generator; to avoid destroying our parent program (if it has already seeded with a specific value for some reason) we shouldn't use the global but instantiate a local source instead... not sure if it makes sense to create a package-local in `sarama.go` or just a real local right here